services/spar/schema/src/V9.hs

```diff
@@ -0,0 +1,38 @@
+-- This file is part of the Wire Server implementation.
+--
+-- Copyright (C) 2020 Wire Swiss GmbH <opensource@wire.com>
+--
+-- This program is free software: you can redistribute it and/or modify it under
+-- the terms of the GNU Affero General Public License as published by the Free
+-- Software Foundation, either version 3 of the License, or (at your option) any
+-- later version.
+--
+-- This program is distributed in the hope that it will be useful, but WITHOUT
+-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+-- details.
+--
+-- You should have received a copy of the GNU Affero General Public License along
+-- with this program. If not, see <https://www.gnu.org/licenses/>.
+
+module V9
+  ( migration,
+  )
+where
+
+import Cassandra.Schema
+import Imports
+import Text.RawString.QQ
+
+migration :: Migration
+migration = Migration 9 "As replacement for `scim_user`, add smaller table for storing time stamps only" $ do
+  void $
+    schema'
+      [r|
+        CREATE TABLE if not exists scim_user_times
+          ( uid             uuid
+          , created_at      timestamp
+          , last_updated_at timestamp
+          , primary key (uid)
+          ) with compaction = {'class': 'LeveledCompactionStrategy'};
+      |]
```

Comment:
maybe it's better to use cassandra write times from brig after all?  can't decide.

Reply:
I had a quick chat with @arianvp about this. It seems like we don't know if it would require any expensive operations in cassandra to be able to write and immediately read the results. Let's try to figure out if we can confidently say it in one way or another and make a decision based on that.

Reply:
https://github.com/zinfra/backend-issues/issues/1631  (i want to do this in a different PR, more important stuff to do now).